Sources/StreamChat/Controllers/ConnectionController/ConnectionController.swift

```diff
@@ -93,8 +93,8 @@ public extension ChatConnectionController {
     /// called with an error.
     ///
     func connect(completion: ((Error?) -> Void)? = nil) {
-        chatClientUpdater.connect { error in
-            self.callback {
+        connectionRepository.connect { [weak self] error in
+            self?.callback {
```

Comment:
I remember before, we could not have weak-self, otherwise, the Updater would not work. This is not the case anymore?

Reply:
Tests were failing because we test that the controller is successfully deallocated, so had to change that.
Still, rest of the tests are passing fine, so no issue there.